react/features/mobile/polyfills/querySelectorPolyfill.js

```diff
@@ -0,0 +1,281 @@
+/**
+ * Parses a CSS selector into reusable components.
+ *
+ * @param {string} selector - The CSS selector to parse.
+ * @returns {Object} - Object with tagName and attrConditions properties.
+ */
+function _parseSelector(selector) {
+    // Wildcard selector
+    if (selector === '*') {
+        return {
+            tagName: null, // null means match all tag names
+            attrConditions: []
+        };
+    }
+
+    // Simple tag name
+    if (/^[a-zA-Z][\w-]*$/.test(selector)) {
+        return {
+            tagName: selector,
+            attrConditions: []
+        };
+    }
+
+    // Attribute selector: tagname[attr="value"] or
+    // tagname[attr1="value1"][attr2="value2"] (with optional wildcard namespace)
+    const multiAttrMatch = selector.match(/^([a-zA-Z][\w-]*)?(\[(?:\*\|)?([^=\]]+)=["']?([^"'\]]+)["']?\])+$/);
```

Comment:
Question: we are creating instances of these regexes every time we create a query, which will be many many times, any chacne caching them helps? something like `const MULTI_ATTR_RE = ...`; at the top and then use that?